easybuild/easyblocks/q/qt.py

```diff
@@ -48,7 +48,12 @@ def extra_options():
         extra_vars = {
              'platform': [None, "Target platform to build for (e.g. linux-g++-64, linux-icc-64)", CUSTOM],
         }
-        return ConfigureMake.extra_options(extra_vars)
+        extra_vars = ConfigureMake.extra_options(extra_vars)
+
+        # allowing to specify prefix_opt doesn't make sense for Qt, since -prefix is hardcoded in configure_step
+        del extra_vars['prefix_opt']
```

Comment:
I'm not sure that's worth the trouble here, since you won't see the `prefix_opt` pop up anywhere (e.g. the output of `eb -a -e EB_Qt`, etc.).

Also, we'd need to create a separate logger for it, since this is a static method (hence no `self.log` available here...). I would rather not to do...